src/app/import/import.component.ts

```diff
@@ -45,72 +47,40 @@ export class ImportComponent {
     private StartStateEnum = StartStateEnum;
     private startState: StartStateEnum;
 
-    private modelPreviewList: ModelPreview[];
+    // The modelPreview that is currently selected
     private selectedModelPreview: ModelPreview = undefined;
 
+    // A list containing all model previews that have been retrieved
+    modelPreviewList: BehaviorSubject<ModelPreview[]> = undefined;
+
     // path file
     pathFile: FileList = undefined;
 
-    // first timing file
+    // track file
+    trackFile: FileList = undefined;
+
+    // Real-timing file
```

Comment:
Inconsistent capitalization

```suggestion
    // real-timing file
```

Reply:
Why was this not approved?